src/style.toit

```diff
@@ -279,35 +279,40 @@ class RoundedCornerBorder extends InvisibleBorder:
     bottom := y2 + h2 - radius_
     if transparency-map is one-byte.OneByteCanvas_:
```

Comment:
I generally prefer to have a field/getter on the object to give me the information.
It would be hard to mock the one-byte-canvas without subclassing it, while a getter could be implemented by anyone.